pkg/definition/gen_sdk/gen_sdk.go

```diff
@@ -414,10 +419,10 @@ func (g *Generator) GenerateCode() (err error) {
 func completeFreeFormSchema(schema *openapi3.SchemaRef) {
 	v := schema.Value
 	if v.OneOf == nil && v.AnyOf == nil && v.AllOf == nil && v.Properties == nil {
-		if v.Type == "object" {
+		if v.Type == openapi3.TypeObject {
```

Comment:
If replace `v.Type == "object"` with `v.Type == openapi3.TypeObject`, how to handle `object`?

Reply:
openapi3.TypeObject is "object" literally.

Reply:
> openapi3.TypeObject is "object" literally.

ok